### PR TITLE
Code for stopping a script

### LINKED ISSUE #144: TBC: make the stop script button work
Make the stop script button work.

It should abort the running script and discard the workspace.

Previous workspace, if any, should be retained.

### PR DESCRIPTION
**Pull request by sunilshetye**
Closes #144

### CODE REVIEW

File: SendLog.py [hunk 1]
```diff
@@ -443,14 +469,44 @@ def uploadscript():
...
         msg = 'Timeout'
         script.status = -4
         rv = {'status': script.status, 'msg': msg}
         return Response(json.dumps(rv), mimetype='application/json')
 
 
+@app.route('/stopscript', methods=['POST'])
+def kill_script(script=None):
+    '''Below route is called for stopping a running script file.'''
+    if script is None:
```
Comment (diptighosalkar) [inline SendLog.py]: I think this check is not needed as in we are checking below

Comment (sunilshetye) [reply SendLog.py]: kill_script is called by two routes: one by the code (where script is not None) and by the /stopscript url (where script is None). Hence this check is required.

Comment (sunilshetye) [reply SendLog.py]: the code is confusing. the lower check needs to be indented. I will fix the code.

Comment (diptighosalkar) [reply SendLog.py]: ok, rest is good to go we can merged it

**Pull request merged**